config/config_test.go

```diff
@@ -331,3 +332,36 @@ func TestConfigSecretsManager_FilterNames(t *testing.T) {
 
 	return
 }
+
+func TestShouldInclude_AllowWhenEmpty(t *testing.T) {
+	var includeREs []Expression
+	var excludeREs []Expression
+
+	assert.True(t, ShouldInclude("test-open-vpn", includeREs, excludeREs),
+		"Should include when both lists are empty")
+}
+
+func TestShouldInclude_ExcludeWhenMatches(t *testing.T) {
+	var includeREs []Expression
+
+	exclude, err := regexp.Compile(`test.*`)
+	require.NoError(t, err)
+	excludeREs := []Expression { { RE: *exclude } }
+
+	assert.False(t, ShouldInclude("test-openvpn-123", includeREs, excludeREs),
+		"Should not include when matches from the 'exclude' list")
+	assert.True(t, ShouldInclude("tf-state-bucket", includeREs, excludeREs),
+		"Should include when doesn't matches from the 'exclude' list")
+}
+func TestShouldInclude_IncludeWhenMatches(t *testing.T) {
```

Comment:
Nice unit tests 👍 

NIT: consider adding one for the case where there are both include and exclude values specified.

Reply:
Added another test at 43bcacbc35e